webpack.config.js

```diff
@@ -2,10 +2,12 @@ var path = require('path');
 var webpack = require('webpack');
 
 var BUILD_DIR = path.resolve(__dirname, 'public/bin');
-var JSX_DIR = path.resolve(__dirname, 'public/jsx');
+var JSX_DIR = path.resolve(__dirname, 'jsx');
```

Comment:
We didn't have to, but it doesn't really make sense to have jsx in the public folder. The public folder is what we tell webpack to serve to the browser, so if we keep the jsx fodler in their, it will be sent to the browser! We only want the bundle being sent to the browser, so you can keep your code in a `src/` folder or some other thing, and then let webpack handle placing the final bundle into the public folder. You don't really need anyone to see or know about the `jsx/` in the browser.